### PR TITLE
Display unknown message warning for messages types we don't understand

### PR DESCRIPTION
**Pull request by typfel**
### Background
When we introduce new messages types in future versions of Wire, we want old clients to display a warning to the user when it receives a a message type it doesn't understand.

needs:
- [x] https://github.com/wireapp/wire-ios-sync-engine/pull/258

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/UnknownMessageCell.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+//
+
+import Foundation
+import Cartography
+import TTTAttributedLabel
+import Classy
+
+public final class UnknownMessageCell : ConversationCell {
+    
+    private let supportURL = URL(string: "http://support.wire.com/error/unknown-message") // TODO update final link
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/UnknownMessageCell.swift]: We have a directory of links in `NSURL+WireURLs.m`

Comment (jenskasemets) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/UnknownMessageCell.swift]: Final link is https://wire.com/compatibility/unknown-message

**Comment by marcoconti83** [comment]
SE merged but not released

**Comment by typfel** [comment]
@mikeger changed a few things. Please re-check.

**Pull request merged**